osh/client/commands/cmd_build.py

```diff
@@ -81,6 +82,8 @@ def check_build(self, args, kwargs, prefix=""):
             if result is not None:
                 self.parser.error(result)
             options[prefix + "brew_build"] = nvr
+        elif dist_git_url is not None:
```

Comment:
`''` also is not `None`

Reply:
No I think `dist_git_url` won't be an empty string here